admin_apps/journeys/iteration.py

```diff
@@ -621,6 +631,11 @@ def set_up_requirements() -> None:
 
     file_name = st.selectbox("File name", options=available_files, index=None)
 
+    experimental_features = st.checkbox(
```

Comment:
We will need to add the same experimental_features checkbox in `set_looker_semantic` in `admin_apps/partner/looker.py`. Currently, following the looker integration workflow causes error: `AttributeError: st.session_state has no attribute "experimental_features". `

If you set `key='experimental_features'` on the checkbox, we should be able to skip passing it back to session_state as the widget will write directly to session_state under the key. 

Everything else looks good and I tested.

Reply:
Ah great catch, added this checkbox to `looker.py`! I don't have the proper credentials to test this (please point me towards them if they're shareable), so lmk if that works.

For some reason the `key` approach in session state wasn't carrying over properly for me, e.g. I saw the session state updating properly with a `key` from within the builder flow but `experimental_features` was still undefined from the iteration flow. This seemed strange to me so I just kept manually setting it for now.